<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ORIR_Debug_Tool</class>
 <widget class="QWidget" name="ORIR_Debug_Tool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="load_pointcloud_data_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>1加载点云图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set_edge_nums_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>2确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>边界点数量设置（默认均为200）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>保存为图片：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_pic_btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>580</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>5保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="translation_pic_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>3平移</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scale_pic_btn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>4缩放</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>211</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>  上</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="up_nums_le">
        <property name="text">
         <string>200</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>  左</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="left_nums_le">
        <property name="text">
         <string>200</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>  下</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="bottom_nums_le">
        <property name="text">
         <string>200</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>  右</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="right_nums_le">
        <property name="text">
         <string>200</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>560</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>图片的宽：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pic_width_le">
      <property name="text">
       <string>700</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>图片的高：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pic_height_le">
      <property name="text">
       <string>400</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>401</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>输出：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="output_te"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
